fudge/__init__.py

```diff
@@ -888,6 +892,25 @@ def has_attr(self, **attributes):
         self._attributes.update(attributes)
         return self
 
+    def has_property(self, **properties):
```

Comment:
``has_properties()`` would read a little better to me since you can pass in multiple